parent/pom.xml

```diff
@@ -153,6 +153,8 @@
 		<!-- discovery.tpc.version usually is usually set to ${tpc.version} but it
 		can happen that they need to have different values, hence 2 properties -->
 		<discovery.tpc.version>${TARGET_PLATFORM_CENTRAL_MAX}</discovery.tpc.version>
+		<testProduct>com.jboss.devstudio.core.product</testProduct>
+		<devstudio.repository>https://devstudio.redhat.com/10.0/snapshots/updates/</devstudio.repository>
```

Comment:
BTW, if any of these properties need to be fed to Maven via bash commandline, I would suggest not using periods in them, as Bash doesn't understand how to deal with a ${variable.name}.

Better to use ${variableName} or ${variable_name} or even ${variablename}.